libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.ts

```diff
@@ -0,0 +1,293 @@
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { MapboxMapService } from '@tamu-gisc/maps/mapbox';
+import { Moment } from 'moment';
+import { forkJoin } from 'rxjs';
+
+@Component({
+  selector: 'tamu-gisc-time-map',
+  templateUrl: './time-map.component.html',
+  styleUrls: ['./time-map.component.scss']
+})
+export class TimeMapComponent implements OnInit {
+  constructor(private mapService: MapboxMapService, private http: HttpClient) {}
+
+  private maxDate: string;
+  private mortalButtonToggled: boolean;
+  private stateButtonToggle: boolean;
```

Comment:
Default value can be assigned here (`false`)

Reply:
resolved